### PR TITLE
bin/rtlcss.js: move require at the top

### PR DESCRIPTION
**Pull request by XhmikosR**
Noticed it in #199.

### CODE REVIEW

File: bin/rtlcss.js [hunk 1]
```diff
@@ -48,8 +49,8 @@ function printHelp () {
   console.log('')
   console.log('*If no destination is specified, output will be written to the same input folder as {source}.rtl.{ext}')
   console.log('')
-  printInfo('RTLCSS version: ' + require('../package.json').version)
-  printInfo('Report issues to: https://github.com/MohammadYounes/rtlcss/issues')
+  printInfo('RTLCSS version: ' + version)
+  printInfo('Report issues to: ' + bugs.url)
```
Comment (XhmikosR) [inline bin/rtlcss.js:53]: Switched to the value from package.json here

**Comment by XhmikosR** [comment]
@MohammadYounes WDYT about this?

**Comment by MohammadYounes** [comment]
@XhmikosR Just waiting for #199 which covers this.

**Comment by XhmikosR** [comment]
I might drop #199, not sure all changes make sense or if you like all of them :) My plan is to merge this separately, and split #199 into separate patches so that we can drop anything you don't like.

For example, as much as I'd like the `strict` check to be on, I can't find a way to enable it without changing the parser options (which in turn makes some of standard's rules moot).

**Pull request merged**